back/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "workadventureback",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "tsc": "tsc",
+    "dev": "ts-node-dev --respawn --transpileOnly ./server.ts",
+    "prod": "tsc && node ./dist/server.js",
+    "lint": "node_modules/.bin/eslint src/ . --ext .ts"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/thecodingmachine/workadventure.git"
+  },
+  "author": "g.parant@thecodingmachine.com",
+  "license": "ISC",
```

Comment:
I'd rather use the "AGPL" license here, which is more restrictive.
"ISC" is very open. It means that someone can take this work, modify it and put it online without redistributing changes.
"AGPL" instead means that anuone can take this work, modify it and put it online, but that any changes done to the application must be redistributed (which I find only fair).